zipkin-lens/babel.config.js

```diff
@@ -0,0 +1,18 @@
+const presets = [
+  [
+    '@babel/env',
+    {
+      targets: {
+        edge: '17',
+        firefox: '60',
+        chrome: '67',
+        safari: '11.1',
+      },
+      useBuiltIns: 'usage',
+      corejs: 3,
```

Comment:
If `useBuiltIns` options is specified, `corejs` options should be specified.
`3` is corejs version which we use.

Please see:
* https://babeljs.io/docs/en/babel-preset-env#corejs